dds/idl/dds_generator.h

```diff
@@ -941,6 +1018,7 @@ class Fields {
 
 private:
   AST_Structure* node_;
+  bool just_keys_;
```

Comment:
Const?

Reply:
I could. I would have to modify the constructor.